plugins/jobs/src/js/JobDelete.js

```diff
@@ -0,0 +1,70 @@
+import * as React from "react";
+import { componentFromStream } from "data-service";
+import { deleteJob } from "#SRC/js/events/MetronomeClient";
+import { Subject } from "rxjs/Subject";
+import "rxjs/add/operator/map";
+import "rxjs/add/operator/switchMap";
+import "rxjs/add/operator/do";
+import "rxjs/add/operator/combineLatest";
+import "rxjs/add/operator/startWith";
+import "rxjs/add/operator/catch";
+
+import JobDeleteModal from "./components/JobDeleteModal";
+
+function errorIsTaskCurrentRunning(errorMsg) {
+  return /stopCurrentJobRuns=true/.test(errorMsg);
+}
+
+function executeDelete({ jobId, stopCurrentJobRuns, onSuccess, errorMsg }) {
```

Comment:
can you please add JSDoc comments describing the types to all of these functions?